src/explore-education-statistics-admin/src/services/preReleaseUserService.ts

```diff
@@ -4,13 +4,27 @@ export interface PreReleaseUser {
   email: string;
 }
 
+export interface PreReleaseInvitePlan {
+  alreadyAccepted?: string[];
+  alreadyInvited?: string[];
+  invitable?: string[];
+}
+
 const preReleaseUserService = {
   getUsers(releaseId: string): Promise<PreReleaseUser[]> {
     return client.get(`/release/${releaseId}/prerelease-users`);
   },
-  inviteUser(releaseId: string, email: string): Promise<PreReleaseUser> {
+  getInvitePlan(
+    releaseId: string,
+    emails: string,
```

Comment:
As mentioned in the backend - would update this so that we send a list of strings rather than the string of email lines. Same with the `inviteUsers` method.